go-controller/pkg/node/ovn_test.go

```diff
@@ -31,10 +34,15 @@ func NewFakeOVNNode(fexec *ovntest.FakeExec) *FakeOVNNode {
 	}
 }
 
-func (o *FakeOVNNode) start(ctx *cli.Context, objects ...runtime.Object) {
+func (o *FakeOVNNode) start(ctx *cli.Context, egressObjects runtime.Object, objects ...runtime.Object) {
 	_, err := config.InitConfig(ctx, o.fakeExec, nil)
 	Expect(err).NotTo(HaveOccurred())
 
+	if egressObjects == nil {
+		o.fakeEgressClient = egressfirewallfake.NewSimpleClientset()
```

Comment:
This is not needed: `egressfirewallfake.NewSimpleClientset` can accept a `nil` argument and do nothing with it

Reply:
done